spec/rspec/rails/matchers/route_to_spec.rb

```diff
@@ -8,53 +8,53 @@ def assert_recognizes(*)
 
   it "provides a description" do
     matcher = route_to("these" => "options")
-    matcher.matches?(:get => "path")
+    matcher.matches?(get: "path")
     expect(matcher.description).to eq("route {:get=>\"path\"} to {\"these\"=>\"options\"}")
   end
 
   it "delegates to assert_recognizes" do
-    expect(self).to receive(:assert_recognizes).with({ "these" => "options" }, { :method=> :get, :path=>"path" }, {})
-    expect({:get => "path"}).to route_to("these" => "options")
+    expect(self).to receive(:assert_recognizes).with({ "these" => "options" }, { method: :get, path: "path" }, {})
+    expect({ get: "path" }).to route_to("these" => "options")
```

Comment:
Now when I look at so many occurences of spaces inside hashes, I would really prefer them not to be, and would rather go with a space inside empty blocks.